src/components/ui/menu/menuGroup.component.tsx

```diff
@@ -147,7 +147,9 @@ export class MenuGroup extends React.Component<MenuGroupProps, State> {
 
   private renderMeasuringGroupedItems = (evaStyle): MeasuringElement => {
     return (
-      <MeasureElement shouldUseTopInsets={ModalService.getShouldUseTopInsets} onMeasure={this.onSubmenuMeasure}>
+      <MeasureElement 
+        shouldUseTopInsets={ModalService.getShouldUseTopInsets}
```

Comment:
I wouldn't say we should pass it to the MeasureElement manually.
Since both component and ModalService are exported from the same framework - why couldn't we just take `shouldUseTopInsets` during the measuring process and skip passing it through the props?

Reply:
@malashkevich